training/train.py

```diff
@@ -51,6 +51,28 @@ def load_data(s3_client):
     return raw_data_dict_
 
 
+def check_style(data_dict):
+    """Check the format of manifest file."""
+    if data_dict.get("ecosystem") == str("maven"):
```

Comment:
if data_dict.get("ecosystem") == "maven"

Reply:
Ok... But the value would be the same right?

Reply:
Yes. You don't need to typecast a string to string :)

Reply:
ok!!